.github/workflows/actionlint.yml

```diff
@@ -45,6 +45,7 @@ jobs:
           rmdir "$HOMEBREW_TAP_REPOSITORY"
           ln -vs "$GITHUB_WORKSPACE" "$HOMEBREW_TAP_REPOSITORY"
 
+          sed -i 's:/bin/bash -e {0}:bash:' .github/workflows/*.y*ml
```

Comment:
Seems to work: https://github.com/Homebrew/homebrew-core/actions/runs/4691171991/jobs/8315279422?pr=128274#step:5:9